lib/slimmer/processors/tag_mover.rb

```diff
@@ -31,24 +31,34 @@ def wrap_node(src, node)
       node
     end
 
+    def confirm_element(node, scripts_for_head, dest_node)
+      if scripts_for_head.any? { |head_script| node.has_attribute?(head_script) }
+        node.parent.name
+      else
+        dest_node
+      end
+    end
+
     def move_tags(src, dest, type, opts)
       comparison_attrs = opts[:keys] || opts[:must_have]
       min_attrs = opts[:must_have] || []
       already_there = dest.css(type).map { |node|
         tag_fingerprint(node, comparison_attrs)
       }.compact
       dest_node = opts[:dest_node] || "head"
-
+      scripts_for_head = opts[:scripts_for_head] || []
       src.css(type).each do |node|
         next unless include_tag?(node, min_attrs) && !already_there.include?(tag_fingerprint(node, comparison_attrs))
```

Comment:
Could we instead change `include_tag?` to ignore elements if they have the attributes set? I think this would simplify the code and avoid making any attempt to move the tags (even if it's just moving them to where they are).

Reply:
Unfortunately any tags that TagMover does not move and not copied across to the destination file - the `include_tag?` method is to make sure certain tags are not malformed/missing requisite information.

For the tags that are filtered by TagMover, it is necessary to perform some manipulation or lose them.